### PR TITLE
add housing affordability map blogpost

### PR DESCRIPTION
**Pull request by gregbunce**
this post includes a map showing housing affordability in utah based on zipcode

### CODE REVIEW

File: src/content/blog/2024-05-17-utah-housing-affordability-map.mdx [hunk 1]
```diff
@@ -0,0 +1,80 @@
+---
+title: 'Mapping Utah’s Housing Affordability'
+author: Greg Bunce
+date: 2024-05-17T15:00:00.000Z
+category: Mapping
```
Comment (steveoh) [inline src/content/blog/2024-05-17-utah-housing-affordability-map.mdx:5]: **issue**: this is not a valid blog category

File: src/content/blog/2024-05-17-utah-housing-affordability-map.mdx [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+cover_image_alt: mapping utah's housing affordability
+---
+
+import Contacts from '@components/page/Contacts.astro';
+
+Utahns are feeling the squeeze, and there's data to back it up. Housing affordability has become the top concern for residents, according to a recent [2024 Utah Foundation survey](https://www.utahfoundation.org/wp-content/uploads/rr819.pdf).
+
+That’s not surprising, given the median sales price of a home in Utah is [$535,000](https://www.redfin.com/state/Utah/housing-market). For context, that’s [27% higher](https://www.redfin.com/us-housing-market) than the national median.
+
+To afford the median mortgage In Salt Lake City, you would need a household income of [at least $140,000](https://www.axios.com/local/salt-lake-city/2023/11/02/affordable-home-slc-house-mortgage-prices), significantly higher than the city’s median of $92,000. These numbers validate that income does not match the cost of housing.
```
Comment (steveoh) [inline src/content/blog/2024-05-17-utah-housing-affordability-map.mdx:19]: **suggestion**: casing

```suggestion
To afford the median mortgage in Salt Lake City, you would need a household income of [at least $140,000](https://www.axios.com/local/salt-lake-city/2023/11/02/affordable-home-slc-house-mortgage-prices), significantly higher than the city’s median of $92,000. These numbers validate that income does not match the cost of housing.
```

File: src/content/blog/2024-05-17-utah-housing-affordability-map.mdx [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+Housing affordability refers to the relationship between the cost of housing and a household's income. It measures how easy or difficult it is for a resident to afford a home.
+
+For this affordability analysis, we’ll be using a home-price-to-income ratio. Home prices have historically had a relatively stable relationship with income, making this ratio a good indicator of affordability.
+
+This ratio is a simple mathematical equation that helps determine housing affordability by dividing the median house price by the median household income. For example, a median house price of $500,000 divided by a median household income of $100,000 equals a home-price-to-income ratio of 5.
+
+Most [experts agree](https://www.fidelity.com/viewpoints/personal-finance/before-buying-house) that a ratio higher than 5 indicates a decline in homebuyer affordability.
+
+Of the 166 Utah ZIP Codes we analyzed for this study, we found that 60% have a home-price-to-income ratio greater than 5.
```
Comment (steveoh) [inline src/content/blog/2024-05-17-utah-housing-affordability-map.mdx:33]:
```suggestion
Of the 166 Utah ZIP codes we analyzed for this study, we found that 60% have a home-price-to-income ratio greater than 5.
```

File: src/content/blog/2024-05-17-utah-housing-affordability-map.mdx [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+
+Here are the 5 least and most affordable ZIP Codes in Utah in terms of home-price-to-income (HPI) ratio.
```
Comment (steveoh) [inline src/content/blog/2024-05-17-utah-housing-affordability-map.mdx:35]:
```suggestion
Here are the 5 least and most affordable ZIP codes in Utah in terms of home-price-to-income (HPI) ratio.
```

File: src/content/blog/2024-05-17-utah-housing-affordability-map.mdx (hunk 1 above)
Comment (gregbunce) [reply src/content/blog/2024-05-17-utah-housing-affordability-map.mdx:5]: do we have an official list of categories?

Comment (gregbunce) [reply src/content/blog/2024-05-17-utah-housing-affordability-map.mdx]: based on the other posts, UGRC seems fitting, aye?

**Review by steveoh — APPROVED** [review]

**Pull request merged**